src/utils.ts

```diff
@@ -168,3 +126,13 @@ export function getConfigValue(value: any, condition: (value: any) => boolean, d
     }
     return result;
 }
+
+export function isBoolean(value: any) {
```

Comment:
@semarche-kaltura  
Can you elaborate why you need isBoolean? 
Are you validating that the  config value is indeed a boolean?
Why?

Reply:
We use this method for checking a config values in case when the config contains inappropriate values like ('yes' instead of true)